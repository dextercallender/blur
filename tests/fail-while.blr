int main() {
        int i;
        i = 5;
        while("sed"){
                println(i);
                i = i - 1;
        }
        println(42);
        return 0;
}
