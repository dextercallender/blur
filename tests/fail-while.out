Fatal error: exception Failure("illogical while")
